<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffMainView</class>
 <widget class="QMainWindow" name="StaffMainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F0F2F5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="icon_only_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #0091FF</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="semester_btn1">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color: #0082E5;
}

QPushButton:checked{
	background-color: #006EDC;
}

QPushButton{
	background-color: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/Asset/staffViewAsset/semester1.png</normaloff>
             <normalon>:/Asset/staffViewAsset/semester2.png</normalon>:/Asset/staffViewAsset/semester1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="class_btn1">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color: #0082E5;
}

QPushButton:checked{
	background-color: #006EDC;
}

QPushButton{
	background-color: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/Asset/staffViewAsset/class1.png</normaloff>
             <normalon>:/Asset/staffViewAsset/class2.png</normalon>:/Asset/staffViewAsset/class1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>347</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exit_btn1">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color: #0082E5;
}

QPushButton{
	background-color: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/asset/loginpageAsset/exit.png</normaloff>
             <normalon>:/asset/loginpageAsset/exit_toggled.png</normalon>:/asset/loginpageAsset/exit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="full_menu_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>172</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>172</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #0091FF</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="semester_btn2">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color: #0082E5;
}

QPushButton:checked{
	background-color: #006EDC;
}

QPushButton{
	background-color: transparent;
	border: none;
	color: white;
	text-align: left;
	padding: 8px 0 8px 25px;
}</string>
           </property>
           <property name="text">
            <string>Semesters</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/Asset/staffViewAsset/semester1.png</normaloff>
             <normalon>:/Asset/staffViewAsset/semester2.png</normalon>:/Asset/staffViewAsset/semester1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="class_btn2">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color: #0082E5;
}

QPushButton:checked{
	background-color: #006EDC;
}

QPushButton{
	background-color: transparent;
	border: none;
	color: white;
	text-align: left;
	padding: 8px 0 8px 25px;
}</string>
           </property>
           <property name="text">
            <string>Classes</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/Asset/staffViewAsset/class1.png</normaloff>
             <normalon>:/Asset/staffViewAsset/class2.png</normalon>:/Asset/staffViewAsset/class1.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>403</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="exit_btn2">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton:hover{
	background-color: #0082E5;
}

QPushButton:checked{
	background-color: #006EDC;
}

QPushButton{
	background-color: transparent;
	border: none;
	color: white;
	text-align: left;
	padding: 8px 0 8px 25px;
}</string>
           </property>
           <property name="text">
            <string>Log out</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/asset/loginpageAsset/exit.png</normaloff>:/asset/loginpageAsset/exit.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="main_view" native="true">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>612</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>660</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="menu_btn">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
	border: none
}

QPushButton:hover{
	background-color: lightgray
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/asset/loginpageAsset/menu.png</normaloff>:/asset/loginpageAsset/menu.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>626</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="stackedWidget">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>400</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>3</number>
             </property>
             <widget class="QWidget" name="page_1">
              <layout class="QGridLayout" name="gridLayout_4"/>
             </widget>
             <widget class="QWidget" name="page_2">
              <layout class="QGridLayout" name="gridLayout_8">
               <property name="leftMargin">
                <number>5</number>
               </property>
               <property name="topMargin">
                <number>5</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>5</number>
               </property>
              </layout>
             </widget>
             <widget class="QWidget" name="page_3">
              <layout class="QGridLayout" name="gridLayout_5"/>
             </widget>
             <widget class="QWidget" name="page">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QLabel" name="course_label">
                     <property name="styleSheet">
                      <string notr="true">text-align: center; font-weight: 500; color: #0D63E6; border: none;</string>
                     </property>
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="backButton">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>27</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton { text-align: center; font-weight: 500; color: #0D63E6; border: 1px solid gray; background-color: white; border-radius: 5px;  }

QPushButton::hover{color: #ff6600; text-decoration: underline;}</string>
                     </property>
                     <property name="text">
                      <string>Back</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QListWidget" name="course_function_list">
                   <property name="styleSheet">
                    <string notr="true">background-color: white;
border: 1px solid lightgray</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>View lists of students in this course</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>View scoreboard of this course</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Update course infomation</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Update a student's result</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Add a student to this course</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Remove a student from this course</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Import list of students to this course</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Export list of students from this course</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Delete this course</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_4">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="class_name">
                     <property name="styleSheet">
                      <string notr="true">text-align: center; font-weight: 500; color: #0D63E6; border: none;</string>
                     </property>
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="backButton2">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>27</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton { text-align: center; font-weight: 500; color: #0D63E6; border: 1px solid gray; background-color: white; border-radius: 5px;  }

QPushButton::hover{color: #ff6600; text-decoration: underline;}</string>
                     </property>
                     <property name="text">
                      <string>Back</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0">
                  <widget class="QListWidget" name="class_function_list">
                   <property name="styleSheet">
                    <string notr="true">background-color: white;
border: 1px solid lightgray</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>Add a student to this class (first year student)</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Import a list of student to this class</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>View list of students in this class</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>View the scoreboard of this class</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="weather-few-clouds">
     <normaloff>:/loginpageAsset/exit.png</normaloff>:/loginpageAsset/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click here to exit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>StaffMainView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_btn2</sender>
   <signal>clicked()</signal>
   <receiver>exit_btn1</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semester_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>semester_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semester_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>semester_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>class_btn1</sender>
   <signal>toggled(bool)</signal>
   <receiver>class_btn2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>class_btn2</sender>
   <signal>toggled(bool)</signal>
   <receiver>class_btn1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
